docs/migrating.md

```diff
@@ -1 +1,84 @@
+| `SIGNALFX_ENDPOINT_URL`                  | The URL to where trace exporters send traces. Overrides `SIGNALFX_REALM` configuration for the traces ingestion endpoint.                                                                             | `http://localhost:9411/api/v2/spans` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`  |
+| `SIGNALFX_METRICS_ENDPOINT_URL`          | The URL to where metric exporters send metrics. Overrides `SIGNALFX_REALM` configuration for the metrics ingestion endpoint.                                                                          | `http://localhost:9943/v2/datapoint` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` |
+| `SIGNALFX_TRACE_PARTIAL_FLUSH_ENABLED`   | Enable to export traces that contain a minimum number of closed spans, as defined by `SIGNALFX_TRACE_PARTIAL_FLUSH_MIN_SPANS`.                                                                        | `false`                              | none                                                                |
+| `SIGNALFX_TRACE_PARTIAL_FLUSH_MIN_SPANS` | Minimum number of closed spans in a trace before it's exported. The default value is ``500``. Requires the value of the ``SIGNALFX_TRACE_PARTIAL_FLUSH_ENABLED`` environment variable to be ``true``. | `500`                                | none                                                                |
+| `SIGNALFX_TRACE_BUFFER_SIZE`             | The size of the trace exporter buffer, expressed as the number of traces.                                                                                                                             | `1000`                               | `OTEL_BSP_MAX_QUEUE_SIZE` (default: 2048)                           |
+
+## Trace propagation settings
+
+| Setting                | Description                                                                                                                                                   | Default   | Otel                                                                               |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|------------------------------------------------------------------------------------|
+| `SIGNALFX_PROPAGATORS` | Comma-separated list of the propagators for the tracer. Available propagators are: `B3`, `W3C`. The Tracer will try to execute extraction in the given order. | `B3,W3C`  | `OTEL_PROPAGATORS`(default: `tracecontext`, `baggage`, available: `b3multi`, `b3`) |
```

Comment:
other available?

b3 should not be listed here: https://github.com/signalfx/gdi-specification/blob/6d04201f3472c83792666d1477aa770027c9a069/specification/configuration.md?plain=1#L214-L216

Lack of information that B3 should be changed to b3multi and W3C to `tracecontext,baggage`.
Lack of information that if the user want to have same default as previous he should make additional Information.

Reply:
The set of values available comes from [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/src/OpenTelemetry.AutoInstrumentation/Configuration/Propagator.cs)
Below is the mapping between `OTEL_PROPAGATORS` and `SIGNALFX_PROPAGATORS` values (comes from [here](https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/advanced-config.md#trace-propagation-settings)).
Not sure why b3 should not be listed - it is supported as evident in the code. The spec states what must be supported, not that b3 mustn't be supported.

Reply:
additional info about configuration required to keep the same propagators configured after migrating from signalfx added in https://github.com/signalfx/splunk-otel-dotnet/pull/124/commits/8b847f975a21de22ff2ec68b8a96cea358758dcb